zebra-rpc/src/methods/types/transaction.rs

```diff
@@ -460,13 +460,21 @@ pub struct ShieldedSpend {
     spend_auth_sig: [u8; 64],
 }
 
+// We can't use `#[getter(skip)]` as upstream `sapling_crypto::note::ValueCommitment` is not `Copy`.
```

Comment:
Yep that's fine as is I think. Just a fix for the comment:

```suggestion
// We can't use `#[getter(copy)]` as upstream `sapling_crypto::note::ValueCommitment` is not `Copy`.
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/9828/commits/4b69d1a52eadfa56bb55f5fbb582b9ca305bc9bb